evgMrmApp/Db/evgMrm.db

```diff
@@ -157,9 +157,29 @@ record(mbbiDirect, "$(P)PpsInp-MbbiDir_") {
   field(ASG, "private")
   field(DESC, "EVG Pps Input")
   field(INP,  "$(P)PpsInp-Sel.RVAL NPP")
+  field(FLNK, "$(P)PpsInp-SQ_")
```

Comment:
I think a race condition exists here. 

For example, currently PpsInp2 is used and one want to set PpsInp, 
the logic here is to first set PpsInp, then clear PpsInp1 and PpsInp2.
During this short period, there are still two Pps sources enabled.